### PR TITLE
Add vlan 50

### PR DESCRIPTION
**Pull request by ntc-training**

### CODE REVIEW

**Comment by ntc-training** [comment]
Add comments, have discussions.

**Comment by davic80** [comment]
I see no issues to add that new vlan

**Comment by nealluo** [comment]
just browsing

File: group_vars/all.yml [hunk 1]
```diff
@@ -7,3 +7,5 @@ vlans:
     name: USERS
   - id: 40
     name: EMAIL
+  - id: 50
+    name: SOMETHING
```
Comment (davic80) [inline group_vars/all.yml:11]: I'd suggest a better name!

Comment (ntc-training) [reply group_vars/all.yml:11]: I know, this is really bad :(

**Pull request merged**